[CWS-4556] Update workload protection docs for multi-policy v2 (#29511)
DataDog/documentation: content/en/security/workload_protection/workload_security_rules/custom_rules.md

Tags identify two things: the Agents using the policy and the infrastructure where those Agents apply the policy. For example, if a policy has the tag `cluster_name:mycluster` the Agents in that cluster use the policy on the hosts in that cluster.

1. Go to [Agent Configuration][3].
2. Open a policy and click **Edit**.
3.  Enter tags and click **Apply**. If the policy is enabled, the policy is applied to the tag targets.

> When you add tags, Datadog displays how many agents the tags target as well as a list of the infrastructure running those agents. For example, `Tags match 144 agents`. 

Reviewer: ```suggestion
When you add tags, Datadog displays how many agents the tags target as well the infrastructure running each agents. For example, `Tags match 144 agents`. 
```
